01week/bio/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <title>Eric Garcia - Bio page</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <header>
+        <div>
+            <div style="background-image: url(https://s24.postimg.org/yii3xmtud/My_Face_S.jpg)" class="face">
+        </div>
+        <div class="name-container">
+            <h1 class="name">Eric "Slash" Garcia</h1>
+            <div class="specialty-badge no-stack">No Stack</div>
+        </div>
+        <h4>Hi fives and good vibes!</h4>
+        <!--List of bands since I don't have anything worth a damn to add here, yet-->
+        <ul class="rockin-list">
+            <li data-toggle="tooltip" data-placement="top" title="Rush">
+                <img src="https://33.media.tumblr.com/ae81de4a2e5586775a735e948ff4bc1e/tumblr_n5zi4yzwOV1r1ms15o1_500.gif" alt="Rush">
+            </li>
+            <li data-toggle="tooltip" data-placement="top" title="Grateful Dead">
+                <img src="https://static1.squarespace.com/static/579fdea8e58c625a66130270/t/57e29ce9b3db2b207f529e4e/1474469113086/" alt="Grateful Dead">
+            </li>
+            <li data-toggle="tooltip" data-placement="top" title="Pink Floyd">
+                <img src="http://media.riffsy.com/images/76bac4879691800855a46fd4c45cafc6/raw" alt="Pink Floyd">
+            </li>
+            <li data-toggle="tooltip" data-placement="top" title="Led Zeppelin">
+                <img src="http://rs194.pbsrc.com/albums/z198/jimmyeightysix/Led%20Zeppelin/LedZeppelin.gif~c200" alt="Led Zeppelin">
+            </li>
+            <li data-toggle="tooltip" data-placement="top" title="The Rolling Stones">
+                <img src="https://68.media.tumblr.com/0144b026a212fe4ce7b0cbfdb43a5ccd/tumblr_ofgys5pdq81v8p6x0o1_500.gif" alt="The Rolling Stones">
+            </li>
+            <li data-toggle="tooltip" data-placement="top" title="Motorhead">
+                <img src="http://cloud-3.steamusercontent.com/ugc/642120405083658103/ED0EDE8C8577A5208DBA8D28E43A6C0F359E97C6/?interpolation=lanczos-none&output-format=jpeg&output-quality=95&fit=inside%7C480:480&composite-to%3D*%2C*%7C480%3A480&background-color=black" alt="Motorhead">
```

Comment:
With long lines like this, try to break them up so they don't go over 80 characters on the line. Just hitting enter in a space wherever it doesn't cut off important information won't change the code itself, but it will make it easier to read for others looking at your code.